cli/src/main.rs

```diff
@@ -147,12 +155,23 @@ async fn store(label: &str, attributes: &impl AsAttributes) -> Result<(), Error>
     Ok(())
 }
 
-async fn lookup(attributes: &impl AsAttributes) -> Result<(), Error> {
+async fn lookup(attributes: &impl AsAttributes, as_hex: bool) -> Result<(), Error> {
     let collection = collection().await?;
     let items = collection.search_items(attributes).await?;
 
     if let Some(item) = items.first() {
-        print_item(item).await?;
```

Comment:
I think we should still use print_item and make it take a as_hex bool.

Reply:
Also not a fan of adding an option to control the output, we should just use hex representation in case the secret is a blob instead of text

Reply:
But hex makes it useless for the purposes of piping into other utils

Reply:
I do use it for this purpose. Most binaries that requiere some sort of identification with a config file allow setting passwords either in plaintext format or by providing a command to execute.  

E.g. mbsync